src/System.Private.CoreLib/src/System/Reflection/Assembly.CoreCLR.cs

```diff
@@ -123,5 +123,8 @@ public static Assembly GetCallingAssembly()
 
         [MethodImpl(MethodImplOptions.InternalCall)]
         internal static extern uint GetAssemblyCount();
+
+        [MethodImpl(MethodImplOptions.InternalCall)]
+        internal static extern bool IsBinderTracingEnabled();
```

Comment:
I am wondering whether AssemblyLoadContext would be a more appropriate place for this. Assembly is a grab bag of everything. AssemblyLoadContext is very specific to assembly loading and presumably quite a bit of the tracing will live on ALC.

Also, this method can be called just `IsTracingEnabled`.